appyters/dimensionality_reduction_visualization/dimensionality_reduction_visualization.ipynb

```diff
@@ -345,14 +441,23 @@
     "    return p"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "caption = \"Leiden 2D scatter plot. Each point presents a column from the input data matrix.\"\n",
+    "rna = leiden_visualization(need_leiden, caption)"
```

Comment:
Please eliminate the `need_leiden` argument in this function and put the if statement outside..